src/viking.js

```diff
@@ -1,11 +1,106 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength) {
+        this.health = health
+        this.strength = strength
+    }
+
+attack() {
+    return this.strength
+}
+receiveDamage(damage) {
+    this.health = this.health - damage
+}
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor (name, health, strength) {
+        super(health, strength)
+        this.name = name
+    }
+    
+receiveDamage(damage) {
+    this.health = this.health - damage
+
+    if (this.health > 0) {
+        return `${this.name} has received ${damage} points of damage`
+    } else {
+        return `${this.name} has died in act of combat`
+    }
+}
+
+battleCry() {
+
+    return "Odin Owns You All!"
+}
```

Comment:
```suggestion
    receiveDamage(damage) {
        this.health = this.health - damage

        if (this.health > 0) {
            return `${this.name} has received ${damage} points of damage`
        } else {
            return `${this.name} has died in act of combat`
        }
    }

    battleCry() {
        return "Odin Owns You All!"
    }
```

Lo mismo de antes!! :D